pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -322,6 +322,9 @@ def required_predicates(self) -> List[Predicate]:
     def default_compilation_pass(self, optimisation_level: int = 1) -> BasePass:
         assert optimisation_level in range(3)
         passlist = [DecomposeBoxes()]
+        # If you make changes to the default_compilation_pass,
+        # then please update this page accordingly
+        # https://cqcl.github.io/pytket-qiskit/api/index.html#default-compilation
```

Comment:
Can you add here the path to the docs file as well?

Reply:
Do you mean like this? `pytket-qiskit/docs/intro.txt` ?

Reply:
yes

Reply:
done, thanks